societes/france.json

```diff
@@ -7951,6 +8050,8 @@
             "name": "Netatmo",
             "url": "http://www.netatmo.com",
             "description": "Smart home",
+            "url": "https://www.netatmo.com",
```

Comment:
this is a duplicate (see the lines above)

Reply:
spotted ! fixed